lib/serverPlugins/modpe.js

```diff
@@ -0,0 +1,336 @@
+var vec3=require("vec3");
+var dir = require("node-dir");
+var fs = require("fs");
+
+module.exports=inject;
+
+function requireFromString(src, filename) {
+    var Module = module.constructor;
+    var m = new Module();
+    m._compile(src, filename);
+    return m.exports;
+}
+
+function log(msg) {
+    console.log("[MODPE-NOINJECT] " + msg);
+}
+
+function modpeApi() {
+    function euclideanMod(numerator, denominator) {
+        var result = numerator % denominator;
+        return result < 0 ? result + denominator : result;
+    }
+
+    function Vec3(x, y, z) {
```

Comment:
Why are you redefining Vec3 here instead of using the vec3 you required at line 1 ?